main.go

```diff
@@ -54,6 +56,44 @@ func sources() []string {
 }
 
 func main() {
+	var (
+		cfgFile        string
+		debug, version bool
+	)
+	flag.StringVar(&cfgFile, "c", "policies.yaml", "Path to configuration file")
+	flag.BoolVar(&version, "v", false, "Print version")
+	flag.BoolVar(&version, "D", false, "Set debug")
+
+	flag.Usage = func() {
+		fmt.Fprintf(os.Stderr, "USAGE: doorman -c policies.yaml\n")
+		fmt.Fprintf(os.Stderr, "\nCLI options\n-----------\n\n")
```

Comment:
you can declare multiple lines string with backticks (see tests with temp ini files)